rebar.lock

```diff
@@ -20,6 +20,18 @@
   {git,"https://github.com/aeternity/aeserialization.git",
       {ref,"80ff8a49f4765db390d6194beae4662e6872ad85"}},
   0},
+ {<<"aesophia">>,
```

Comment:
Let's not put a random commit from the `master/iris` branch of `aesophia` in the aeternity node...